core/src/main/java/hudson/model/ViewJob.java

```diff
@@ -236,14 +236,23 @@ private boolean terminating() {
         @Override
         public void run() {
             while (!terminating()) {
+                String jobName = null;
                 try {
-                    getNext()._reload();
+                    var next = getNext();
+                    jobName = next.getFullName();
+                    next._reload();
+                    jobName = null;
                 } catch (InterruptedException e) {
                     // treat this as a death signal
                     return;
-                } catch (Throwable t) {
+                } catch (Exception e) {
                     // otherwise ignore any error
-                    t.printStackTrace();
+                    if (jobName != null) {
+                        var finalJobName = jobName;
+                        LOGGER.log(Level.FINEST, e, () -> "Failed to reload job " + finalJobName);
+                    } else {
+                        LOGGER.log(Level.FINEST, e, () -> "Failed to obtain next job in the reload queue");
```

Comment:
```suggestion
                        LOGGER.log(Level.WARNING, e, () -> "Failed to reload job " + finalJobName);
                    } else {
                        LOGGER.log(Level.WARNING, e, () -> "Failed to obtain next job in the reload queue");
```
probably (I doubt these are expected conditions)